src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -53,6 +64,18 @@ public static class SymbolDisplay
             return ToMinimalDisplayParts(symbol, semanticModel, position, format).ToDisplayString();
         }
 
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
+        public static string ToMinimalDisplayString(
+#pragma warning restore RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
I dislike this style of of suppression. It makes it much harder to read the signatures. I would suggest we either:

1. Put the `pragma` around the entire method signature, not the name
1. Just put it at the top of the file. Is there any specific value in having this on every API when our response will always be to disable it?

Reply:
I'll put it around the signature. To be entirely honest I'm not at all sure why this is triggering, as the ones it's complaining about have differing numbers of required parameters that are not convertible to each other.

---
In reply to: [259051642](https://github.com/dotnet/roslyn/pull/33531#discussion_r259051642) [](ancestors = 259051642)